webserver/views/user.py

```diff
@@ -55,6 +55,18 @@ def profile(user_name):
 
     user = _get_user(user_name)
 
+    need_exact = request.args.get("exact")
+    if not need_exact:
+        listen_count = db_conn.get_listen_count_for_user(user_name)
```

Comment:
In the listenstore you catch the exception and then pass it on via Raise. However, here you do not catch the exception. You should catch the exception, but since this is a minor aspect of this page, perhaps show an error message when the count cannot be loaded in time. Then the rest of the page can still be rendered, rather than failing the whole page.